src/test/java/hibernate/binder/AnnotationBinderTest.java

```diff
@@ -0,0 +1,33 @@
+package hibernate.binder;
+
+import hibernate.binder.entity.Entity1;
+import hibernate.binder.entity.Entity2;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AnnotationBinderTest {
+
+    @Test
+    void Entity_어노테이션이_달린_클래스를_가져온다() {
+        List<Class<?>> actual = AnnotationBinder.parseEntityClasses("hibernate.binder");
+        assertThat(actual).containsOnly(Entity1.class, Entity2.class, TestEntity.class);
+    }
+
+    @Test
+    void Entity_어노테이션이_있는_클래스가_없는_경우_빈리스트를_반환한다() {
+        List<Class<?>> actual = AnnotationBinder.parseEntityClasses("hibernate.binder.empty");
+        assertThat(actual).isEmpty();
+    }
+
+    @Entity
+    static class TestEntity {
```

Comment:
```suggestion
    private static class TestEntity {
```

AnnotationBinder 의  테스트에만 쓰이는 엔티티를 선언해주시는 점 좋습니다. 👍 
만약 재활용되지 않을 클래스라면 private 접근 제어자를 붙이면 어떨지 제안해봅니다.